package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- You did a great job by using webpack in your project but make sure  that 
1. All webpack-related packages should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as devDependencies
1. Has `npm run ?` scripts like build, start in `package.json`. See the example below.

```json
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```

Please make sure after implementing the point I listed above for the configuration of webpack. you must be able to open the project with using the command npm run start in your terminal. :+1: